unyt/tests/test_array_functions.py

```diff
@@ -408,6 +411,18 @@ def test_histogram():
     assert type(counts) is np.ndarray
     assert bins.units == arr.units
 
+    counts, bins = np.histogram(
```

Comment:
one general comment on this test module: I would prefer that new, independent test cases be added as separate test functions